cluster-operator/image.yaml

```diff
@@ -21,26 +21,26 @@ packages:
 
 modules:
   repositories:
-    - name: cluster-controller
-      path: modules/cluster-controller
+    - name: cluster-operator
+      path: modules/cluster-operator
     - name: common.java
       path: ../modules/common/java
   install:
-    - name: cluster-controller
+    - name: cluster-operator
     - name: common.java
 
 artifacts:
-  - path: cluster-controller-0.3.1.redhat-1.jar
+  - path: cluster-operator-0.3.1.redhat-1.jar
```

Comment:
now that we are on the way for a new release, changing this file name means breaking the dev preview I guess because the `cluster-operator-0.3.1readhat-1.jar` file doesn't exist. What's the way to deal with it?

Reply:
Should we update to something like a 0.6.0?